controllers/machinedeletionremediation_controller.go

```diff
@@ -38,9 +39,10 @@ import (
 )
 
 const (
-	machineAnnotationOpenshift = "machine.openshift.io/machine"
-	machineKind                = "Machine"
-	machineSetKind             = "MachineSet"
+	machineAnnotationOpenshift     = "machine.openshift.io/machine"
+	machineKind                    = "Machine"
+	machineSetKind                 = "MachineSet"
+	maxAllowedErrorToDeleteMachine = 5
```

Comment:
Is there any reason why five is **good* number for max retries for machine deletion?

Reply:
I replied below, together with the other question